extensions/wikia/CreateNewWiki/js/WikiBuilder.js

```diff
@@ -130,9 +145,29 @@ define('ext.createNewWiki.builder', ['ext.createNewWiki.helper'], function (help
 				callback: function (res) {
 					// check phalanx result
 					if (res.msgHeader) {
+						track({
+							action: tracker.ACTIONS.ERROR,
+							label: 'wiki-description-phalanx-validation'
+						});
 						$.showModal(res.msgHeader, res.msgBody);
 						descWikiNext.attr('disabled', false);
 					} else {
+						track({
+							action: tracker.ACTIONS.SUCCESS,
+							label: 'wiki-description-phalanx-validation'
+						});
+						if (descriptionVal === WikiBuilderCfg.descriptionplaceholder) {
+							track({
+								action: tracker.ACTIONS.SUBMIT,
+								label: 'wiki-description-empty'
+							});
+						} else {
+							track({
+								action: tracker.ACTIONS.SUBMIT,
+								label: 'wiki-description-not-empty'
+							});
+						}
```

Comment:
what about
`label = descriptionVal === WikiBuilderCfg.descriptionplaceholder ? 'wiki-description-empty' : 'wiki-description-not-empty';

track({
    action: tracker.ACTIONS.SUBMIT,
    label: label
});
`